### DISCUSSION TITLE
SO moved to airgap: custom rules fails

### DISCUSSION THREAD

**Discussion by nebra-status** [Security-Onion-Solutions/securityonion#13522] [2.4]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

64

### RAM

256GB

### Storage for /

300GB

### Storage for /nsm

12TB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

We moved our system from internet connection to airgap.
The problem we're facing now is to add custom rules:
- we added the custom rule in the customrulesets, refering to the .rules file under custom_file
- updating the rules under Detections->Suricata gave a sync error
- running so-rule-update worked, apart from the internet connection error, and the rules were added to /opt/so/rules/nids/suri/all.rules
- however, this isn't reflected in the Detections dashboard, where the total number of Suricata rules is still the same and the rules added to all.rules aren't found under the Detection dashboard
-  trying to re-launch Suricata full update or differential update gives still the same sync error
- there's no entry in /opt/so/log/soc/sync.log or sensorino-server.log regarding the sync error

Anything else we can do to force the custom rules?

Thanks :)

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment]
When you moved it to airgap did you set it to airgap mode?
